src/widgets/settingspages/ExternalToolsPage.cpp

```diff
@@ -60,6 +69,12 @@ ExternalToolsPage::ExternalToolsPage()
                                  getSettings()->streamlinkUseCustomPath);
         groupLayout->setWidget(2, QFormLayout::SpanningRole, customPathCb);
 
+        auto note = new QLabel("Chatterino expects the binary to be called ");
+        note->setText(note->text().append(getBinaryName()));
```

Comment:
```suggestion
        auto note = new QLabel(QStringLiteral("Chatterino expects the binary to be called \"%1\".").arg(STREAMLINK_BINARY_NAME);
```

We can set the text correctly from the start.